packages/utils/src/saml/bulk-saml.ts

```diff
@@ -0,0 +1,47 @@
+// This is a helper script that lets you send xcpd requests in bulk
+import fs from "fs";
+
+import { createAndSignBulkXCPDRequests } from "@metriport/core/external/saml/xcpd/iti55-envelope";
+import { sendSignedRequests } from "@metriport/core/external/carequality/ihe-gateway-v2/saml-client";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const privateKey = getEnvVarOrFail("IHE_STAGING_KEY");
+const publicCert = getEnvVarOrFail("IHE_STAGING_CERT");
+const certChain = getEnvVarOrFail("IHE_STAGING_CERT_CHAIN");
+const privateKeyPassword = getEnvVarOrFail("IHE_STAGING_KEY_PASSWORD");
+
+// path to a file containing an XCPD request json with many gateways
+const bulkRequestPath = "";
+const bulkData = JSON.parse(fs.readFileSync(bulkRequestPath, "utf8"));
+
+async function bulkXcpdRequests() {
+  const signedRequests = createAndSignBulkXCPDRequests(
+    bulkData,
+    publicCert,
+    privateKey,
+    privateKeyPassword
+  );
+  fs.writeFileSync("../../scratch/outbound_xcpd.xml", signedRequests[0].signedRequest);
```

Comment:
We've been using the `runs/` folder for outputs, check an example on `get-consolidated.ts`